vuu-ui/packages/vuu-utils/test/screenshot-utils.test.ts

```diff
@@ -1,38 +1,38 @@
-
-    // We need to mock the html-to-image package because the web API operations it relies on (e.g. canvas.toDataUrl) are not available in the test environment
```

Comment:
I don't think we need to justify why we are mocking a dependency in this unit test as that is the common practice to do

Reply:
You're right. I think I was getting mixed up trying to make a unit-test-integration-test crossover.